layers/+emacs/org/packages.el

```diff
@@ -918,7 +919,8 @@ Headline^^            Visit entry^^               Filter^^                    Da
         "rI" 'org-roam-insert-immediate
         "rl" 'org-roam-buffer-toggle-display
         "rta" 'org-roam-tag-add
-        "rtd" 'org-roam-tag-delete))
+        "rtd" 'org-roam-tag-delete
+        "rta" 'org-roam-alias-add))
```

Comment:
Did you mean to have the `t` here? From the rest of your PR (and the mnemonics of it), that looks like a mistake.

Reply:
No that t is not supposed to be there. Not sure how it got there, but it should be fixed now.